server/configuration.go

```diff
@@ -203,36 +202,24 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(err, "failed to ensure demo user")
 	}
 
+	path, err := p.API.GetBundlePath()
+	if err != nil {
+		return errors.Wrap(err, "failed to get bundle path")
+	}
+
+	iconImagePath := filepath.Join(path, "/assets/github.svg")
+
 	botId, ensureBotError := p.Helpers.EnsureBot(&model.Bot{
 		Username:    "demoplugin",
 		DisplayName: "Demo Plugin Bot",
 		Description: "A bot account created by the demo plugin.",
-	})
+	}, plugin.IconImagePath(iconImagePath))
```

Comment:
The argument of `plugin.IconImagePath` is a relative path inside the bundle, right? Hence you don't need https://github.com/mattermost/mattermost-plugin-demo/pull/76/files#diff-97fcb5163c5df39ace07e8d58616214eR205-R210.

Reply:
@hanzei Yes, sorry, you're right. I'll update this.